src/dbus/server/dbus_object.cpp

```diff
@@ -134,7 +134,8 @@ void DBusObject::GetPropertyMethodHandler(DBusRequest &aRequest)
     DBusMessageIter iter;
     std::string     interfaceName;
     std::string     propertyName;
-    otError         error = OT_ERROR_NONE;
+    otError         error      = OT_ERROR_NONE;
+    otError         replyError = OT_ERROR_NONE;
```

Comment:
I think we don't need a separate variable here.

Reply:
We need to differentiate between two cases:

- We failed when trying to call the handler, without actually calling the handler
- We called the handler, but the handler returned an OT error

We can't use error values because we don't know what errors the handler may return.

Reply:
But the handling of 2 errors are almost identical. The only difference is the log level "info" vs "warning". Why don't make them all "warning"?

Reply:
If the property path is invalid, this is clearly a logic error on the client side that should not happen. So, such case should use WARN because it's incorrect behavior. 

If the property handler replies error, like `ActiveDatasetTlvs`, it's a normal case not a logical error, so we should use INFO ?